CONTRIBUTING.md

```diff
@@ -48,6 +48,32 @@ We use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
 
 We use [`tap`](https://node-tap.org/) for testing & expect that every new feature or bug fix comes with corresponding tests that validate the solutions. Tap also reports on code coverage and it will fail if that drops below 100%.
 
+## Run npm/npx Locally
+
+If a specific command is not covered by tap, try the following:
```

Comment:
I think this part can be removed too.  The advice is good for general testing if anything you're doing locally when developing, even long before you get to tests.  Everything else looks good!

Reply:
Okay, just removed that section, thanks for all your help!